static/js/subs.js

```diff
@@ -868,10 +868,11 @@ exports.open_create_stream = function () {
 
 
 exports.sub_or_unsub = function (sub, row_object) {
+    const stream_row = $("div[data-stream-id='" + sub.stream_id + "']");
```

Comment:
This selector is way too generic; we have lots of divs with a `data-stream-id` in the webapp codebase.  We want to find the one that's being referenced here, which is tricky, because this is actually called from several places with different types of `row_object` elements.  

```
(zulip-py3-venv) tabbott@coset:~/zulip$ git grep sub_or_unsub
frontend_tests/node_tests/compose.js:        subs.sub_or_unsub = function () {
static/js/compose.js:        subs.sub_or_unsub(sub);
static/js/stream_edit.js:        subs.sub_or_unsub(sub);
static/js/stream_edit.js:        subs.sub_or_unsub(sub, e.currentTarget);
static/js/stream_popover.js:        subs.sub_or_unsub(sub);
static/js/subs.js:        exports.sub_or_unsub(row_data.object);
static/js/subs.js:exports.sub_or_unsub = function (sub, row_object) {
static/js/subs.js:        exports.sub_or_unsub(sub);
```